__tests__/server.test.ts

```diff
@@ -111,6 +118,8 @@ describe('query healthcareProfessionalById', () => {
         expect(errors[0].extensions.http).toEqual({status: 404})
         expect(errors.length).toBe(1)
         expect(errors[0].extensions.code).toBe('NOT_FOUND')
-        expect(errors[0].message).toBe('Healthcare professional not found.')
+        expect(errors[0].message).toBeDefined()
+        expect(errors[0].message.length).toBeGreaterThan(0)
+        expect(spyCustomeErrors).toHaveBeenCalled()
```

Comment:
Are you planning to check the message back to the user in this PR or in a future one?

Reply:
So, this fix was because of issue #255 not passing. I initially believed that my error messages should be more dynamic. However, after reading your feedback (in PR that resolves #255), I'm starting to think that this might not have been necessary at all.

My question now is, what is the best way to test for error messages: as it was done before, or a combination of both?

Reply:
hmmm...I think that since these are endpoint/end-to-end/api/whateveryoucallthem tests, we should just test what is returned back to the user and not take the spy approach